### DISCUSSION TITLE
トラップ感知のロッド　確定入手手段について

### DISCUSSION THREAD

**Discussion by tanguband** [hengband/hengband#3106] [Closed]
序盤の浅い階層で入手出来なかった場合は、
途中なかなか出ない場合も多く不便な探索が続く場合がある

巻物や杖も常時店売りしているわけではないため代替手段となりづらい

序盤に確定入手できる手段があれば、より探索が便利になるのでは？

魔法やレイシャルの感知手段や証人報酬の警告の指輪との兼ね合いもあるので、
そのあたりのバランスは要検討

現在アリーナの対戦で『らいおんはーと』の報酬枠が空いているため
その報酬にするのはどうだろうか？

**Comment by tanguband** [comment]
アイデアを採用していただいたのでクローズします
ありがとうございました

**Comment by habu1010** [comment]
カテゴリだけCloseになって、状態はOpenのままのようなのでクローズしておきます。
